src/app/home-module/advertisment-details/advertisment-details.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-advertisment-details',
+  templateUrl: './advertisment-details.component.html',
+  styleUrls: ['./advertisment-details.component.css']
+})
+export class AdvertismentDetailsComponent implements OnInit {
+  public itemtitle = "Maruti Suzuki Alto 800 Lxi, 2012, Petrol";
```

Comment:
this is bad approach.
Coordinate with Abhishek, I have given same comments on his component.
There should be ad class & same for binding 
Also coordinate with madhu to add service support in your component.